AnaProd/anaTupleProducer.py

```diff
@@ -273,7 +277,7 @@ def createAnatuple(inFile, outDir, config, sample_name, anaCache, snapshotOption
             for puIDbranch in puIDbranches:
                 if puIDbranch in dfw.df.GetColumnNames():
                     new_branch_name= puIDbranch.strip("_tmp")
-                    dfw.DefineAndAppend(new_branch_name, f"{puIDbranch}[ExtraJet_B1]")
+                    dfw.Define(new_branch_name, f"{puIDbranch}[ExtraJet_B1]")
```

Comment:
In this way it will start with `weight_`, isn't it? we want it to start with `ExtraJet_weight_`.